stdlib/knative/src/main/ballerina/src/knative/annotation.bal

```diff
@@ -0,0 +1,250 @@
+#
+# + fieldPath - Path of the field
+public type FieldValue record{|
+    string fieldPath;
+
+|};
+# Value for a secret key.
+#
+# + name - Name of the secret.
+# + key - Key of the secret.
+public type SecretKeyValue record {|
+    string name;
+    string key;
+
+|};
+# Value for config map key.
+#
+# + name - name of the config.
+# + key - key of the config.
+public type ConfigMapKeyValue record {|
+    string name;
+    string key;
+
+|};
+# Value for resource field.
+#
+# + containerName - Name of the container.
+# + resource - Resource field
+public type ResourceFieldValue record {|
+   string containerName?;
+   string 'resource;
+
+|};
+# Value from config map key.
+#
+# + configMapKeyRef - Reference for config map key.
+public type ConfigMapKeyRef record {|
+    ConfigMapKeyValue configMapKeyRef;
+
+|};
+# Value from field.
+#
+# + fieldRef - Reference for a field.
+public type FieldRef record {|
+    FieldValue fieldRef;
+
+|};
+# Value from secret key.
+#
+# + secretKeyRef - Reference for secret key.
+public type SecretKeyRef record {|
+    SecretKeyValue secretKeyRef;
+
+|};
+# Value from resource field.
+#
+# + resourceFieldRef - Reference for resource field.
+public type ResourceFieldRef record {|
+    ResourceFieldValue resourceFieldRef;
+
+|};
+public const string IMAGE_PULL_POLICY_IF_NOT_PRESENT = "IfNotPresent";
+public const string IMAGE_PULL_POLICY_ALWAYS = "Always";
+public const string IMAGE_PULL_POLICY_NEVER = "Never";
+# Image pull policy type field for kubernetes deployment and jobs.
+public type ImagePullPolicy "IfNotPresent"|"Always"|"Never";
+# Probing configuration.
+#
+# + port - Port to check for tcp connection.
+# + initialDelayInSeconds - Initial delay for pobing in seconds.
+# + periodSeconds - Interval between probes in seconds.
+public type ProbeConfiguration record {|
+    int port?;
+    int initialDelayInSeconds?;
+    int periodSeconds?;
+
+|};
+# Type of operations between key and value of a toleration.
+public type TolerationOperator "Exists"|"Equal";
+# Types of toleration effects for pods.
+public type TolerationEffect "NoSchedule"|"PreferNoSchedule"|"NoExecute";
+# Pod toleration configuration.
+#
+# + key - Taint key of the toleration.
+# + operator - Operator between the key and value. Default is `"Equal"`.
+# + value - Taint value of the toleration.
+# + effect - The taint effect
+# + tolerationSeconds - Time period of toleration in seconds. Default is `0`.
+public type PodTolerationConfiguration record {|
+    string key;
+    TolerationOperator operator = "Equal";
+    string value;
+    TolerationEffect effect?;
+    int tolerationSeconds = 0;
+
+|};
+# Knative service configuration.
+#
+# + dockerHost - Docker host IP and docker PORT. ( e.g minikube IP and docker PORT).
+# Default is to use DOCKER_HOST environment variable.
+# If DOCKER_HOST is unavailable, use `"unix:///var/run/docker.sock"` for Unix or use `"npipe:////./pipe/docker_engine"` for Windows 10 or use `"localhost:2375"`.
```

Comment:
Break the line at 120 characters to improve readability. Check all the places.

Reply:
line break added,